test/index.js

```diff
@@ -42,11 +42,12 @@ test('resume', t => {
         resume: true
       }, err => {
         t.error(err)
+        t.equal(status.fileCount, 2)
```

Comment:
This test ended up a little weird because the new file added to the hyperdrive isn't counted. I could rewrite this to update the file on the fs and then resume.